fix(LIC): clarify how forwarding works (#11138)
newrelic/docs-website: src/content/docs/logs/logs-context/configure-logs-context-go.mdx

    title="Decorate and forward logs with APM"
  >

For most users, using the automatic log forwarding built into the Go agent is the best solution. It's easy to set up and works great for a majority of use cases. [Learn about log forwarding limitations.](/docs/logs/logs-context/get-started-logs-context#forwarding-details)

> Automatic application log forwarding is now enabled by default in Go agent v3.20.0+ . If your applications use these Go agent versions, you don't need to change your config options to enable automatic log forwarding.

Reviewer: ```suggestion
Automatic application log forwarding is now enabled by default in Go agent version 3.20.0 or higher. If your applications use these Go agent versions, you don't need to change your config options to enable automatic log forwarding.
```

> Automatic application log forwarding is now enabled by default in versions v3.20.0+ of the Go agent. Applications using these versions of the Go agent do not need to change their config options to enable automatic log forwarding.

Reviewer: ```suggestion
Automatic application log forwarding is now enabled by default in Go agent v3.20.0+ . If your applications use these Go agent versions, you don't need to change your config options to enable automatic log forwarding.
```

> If you're using an older version of the Go agent, modify its config to enable application log forwarding:

Reviewer: ```suggestion
If you're using an older version of the Go agent, you'll need to modify its config to enable application log forwarding:
```

  </Collapser>
</CollapserGroup>

> ## Installing a logs in context plugin

Reviewer: ```suggestion
## Installing a logs-in-context plugin
```
